core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -104,14 +104,11 @@ oppia.controller('CollectionPlayer', [
     };
 
     $scope.getNonRecommendedCollectionNodes = function() {
-      var displayedExplorationIds = [];
+      var displayedExplorationIds = (
+        $scope.collectionPlaythrough.getCompletedExplorationIds());
       if ($scope.collectionPlaythrough.getNextExplorationId() !== null) {
-        displayedExplorationIds = (
-          [$scope.collectionPlaythrough.getNextExplorationId()].concat(
-            $scope.collectionPlaythrough.getCompletedExplorationIds()));
-      } else {
-        displayedExplorationIds = (
-          $scope.collectionPlaythrough.getCompletedExplorationIds());
+        displayedExplorationIds = displayedExplorationIds.append(
```

Comment:
Did you test this?

Reply:
I had done a search, and the function getNonRecommendedCollectionNodes() is not called anywhere. 
Yeah, I just saw that it was supposed to be displayedExplorationIds.push(), not append(). Still, is that function needed, as its not used anywhere?